pkg/validators/basic_validators.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2023 Google LLC
```

Comment:
This being a new file, I would recommend against naming it basic_validators.  The word basic doesn't mean much in this context. Are these all the metadata related validators?

Reply:
ah, also copyright 2023 -> 2025?